### PR TITLE
Update  dtsi file of  STM32H750 by including stm32h743.dtsi 

### PR DESCRIPTION
**Pull request by djiatsaf-st**
The difference between the two files it's only on the values of usbotg_fs node properties.

### CODE REVIEW

**Review by erwango — COMMENTED** [review]
Thanks. 

Did you check that difference is valid ? 
Only difference I see is `wkup` interrupt definition, but is it actually used ?

File: dts/arm/st/h7/stm32h750.dtsi [hunk 1]
```diff
@@ -5,113 +5,20 @@
...
-		};
-
-		dmamux2: dmamux@58025800 {
-			dma-requests= <107>;
-		};
-
 		usbotg_fs: usb@40080000 {
-			compatible = "st,stm32-otgfs";
-			reg = <0x40080000 0x40000>;
+
```
Comment (mathieuchopstm) [inline dts/arm/st/h7/stm32h750.dtsi:16]: Nit: remove newline
```suggestion
```

**Review by erwango — APPROVED** [review]

**Review by mathieuchopstm — APPROVED** [review]

**Review by gautierg-st — APPROVED** [review]

File: dts/arm/st/h7/stm32h750.dtsi [hunk 2]
```diff
@@ -5,113 +5,13 @@
  */
 
 #include <st/h7/stm32h7.dtsi>
+#include <st/h7/stm32h743.dtsi>
```
Comment (erwango) [inline dts/arm/st/h7/stm32h750.dtsi:8]: @djiatsaf-st I've missed this. Please remove `#include <st/h7/stm32h7.dtsi>` as included by `#include <st/h7/stm32h743.dtsi>`

Comment (djiatsaf-st) [reply dts/arm/st/h7/stm32h750.dtsi:8]: Here you can find the associated PR:

https://github.com/zephyrproject-rtos/zephyr/pull/78052

**Pull request merged**